server/commiter.go

```diff
@@ -167,3 +178,34 @@ func (p *Plugin) updateCommitersPost(post *model.Post, userID, org, repo string,
 		return
 	}
 }
+
+func (p *Plugin) verifyOrg(owner string) (bool, error) {
+	_, _, err := p.client.Organizations.Get(context.Background(), owner)
+	if err == nil {
+		return true, nil
+	}
+	_, _, err = p.client.Users.Get(context.Background(), owner)
+	if err == nil {
+		return false, nil
+	}
+
+	return true, fmt.Errorf("Unable to find Github Organization, or User with matching owner: %s", owner)
```

Comment:
sorry for the grammar nitpicks :) 
```suggestion
	return true, fmt.Errorf("Unable to find GitHub Organization or User with matching owner: %s", owner)
```